From 5ff0b966a31575985fa2c1409e8218f337391a8e Mon Sep 17 00:00:00 2001
From: Sergey Senozhatsky <senozhatsky@chromium.org>
Date: Mon, 2 Mar 2020 17:52:05 +0900
Subject: [PATCH 4/4] CHROMIUM: config: Enable Intel camera drivers for dedede

This CL selects requied Intel camera drivers for dedede.

BUG=b:149068439, b:149068672
TEST=Sanity checked basic camera functions.

Change-Id: Ib5d868a4ae27c42cbc4816d8c645cdfd99712176
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
---
 .../config/x86_64/chromeos-intel-pineview.flavour.config   | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config b/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
index 300a88de9917..22744014a66e 100644
--- a/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
+++ b/chromeos/config/x86_64/chromeos-intel-pineview.flavour.config
@@ -126,3 +126,10 @@ CONFIG_WILCO_EC_TELEMETRY=m
 CONFIG_X86_INTEL_LPSS=y
 # CONFIG_X86_MCE_AMD is not set
 CONFIG_X86_PKG_TEMP_THERMAL=y
+CONFIG_VIDEO_INTEL_IPU=m
+CONFIG_VIDEO_OV8856=m
+CONFIG_VIDEO_OV5675=m
+CONFIG_VIDEO_OV9734=m
+CONFIG_VIDEO_DW9768=m
+CONFIG_EEPROM_AT24=m
+CONFIG_VIDEO_INTEL_IPU6SE=y
-- 
2.17.1

